src/Components/App/App.js

```diff
@@ -12,21 +15,25 @@ class App extends Component {
     super(props);
   }
 
-  // componentDidMount = async () => {
-  // const bookList = await getBooks()
-  // const allBooks = bookList.results.books
-  // this.props.getInitialBooks(allBooks)
-  // // .catch(error => console.log('error', error))
-  // // refactor to set user message into state depending on where state will be stored
-  //
-  // }
-
   render() {
     return (
       <main>
         <h1 className='title'>BiblioGoal</h1>
-        <Search />
-        <BookContainer />
+      <nav>
```

Comment:
We will need to add some conditional logic here for displaying the home link since we don't want it to display on the home page but we do want it to display on the Have Read page & the To Read page. I created an issue for this.